test/runtests.jl

```diff
@@ -108,3 +108,9 @@ close(output)
 if prev_theme !== nothing
     ENV["GADFLY_THEME"] = prev_theme
 end
+
+if !haskey(ENV, "TRAVIS") && !isinteractive() &&
+            !isempty(readdir(joinpath((@__DIR__),"cachedoutput"))) &&
+            !isempty(readdir(joinpath((@__DIR__),"gennedoutput")))
+    run(`$(Base.julia_cmd()) $(joinpath(@__DIR__, "compare_examples.jl"))`)  # `include`ing causes it to hang.
+end
```

Comment:
this looks like an oversight in a rebase.  we don't want this here because it causes problems with PkgEval, which does *not* have the TRAVIS environment variable set.  when testing locally, developers will simply have to run the regression tests manually.

Reply:
Indeed. I've cleaned this up now.